Source/Tests/Trakt.NET.Modules.Tests/TraktEpisodesModule/TraktEpisodesModule_GetEpisodeStream_Tests.cs

```diff
@@ -0,0 +1,170 @@
+﻿namespace TraktNet.Modules.Tests.TraktEpisodesModule
+{
+    using FluentAssertions;
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Threading.Tasks;
+    using Trakt.NET.Tests.Utility;
+    using Trakt.NET.Tests.Utility.Traits;
+    using TraktNet.Exceptions;
+    using TraktNet.Objects.Get.Episodes;
+    using TraktNet.Responses;
+    using TraktNet.Extensions;
+    using Xunit;
+
+    [TestCategory("Modules.Episodes")]
+    public partial class TraktEpisodesModule_Tests
+    {
+        private readonly string GET_EPISODE_STREAM_URI = $"shows/{SHOW_ID}/seasons/{SEASON_NR}/episodes/{EPISODE_NR}";
+
+        [Fact]
+        public async Task Test_TraktEpisodesModule_GetEpisodesStreamAsync()
```

Comment:
Test method names should be consistent with the other test method names, meaning that there should not be an "Async" suffix in the names.